README.it.md

```
@@ -0,0 +1,502 @@
+
+- **Il riconoscimento vocale è scarso/lento.**
+→ Usa STT cloud. Su Intel, usa solo i core ad alte prestazioni.
+
+- **Come vengono gestiti i dati?**
+→ La voce e il contenuto della conversazione sono archiviati localmente e non inviati a server.
+
+### [📥 Scarica](https://github.com/kapitalismho/PuriPuly-heart/releases/latest)
+
+---
+
+## Confronto traduzione
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)